docs/security-considerations.rst

```diff
@@ -7,28 +7,28 @@ Security Considerations
 While it is usually quite easy to build software that works as expected,
 it is much harder to check that nobody can use it in a way that was **not** anticipated.
 
-In Solidity, this is even more important because you can use smart contracts
-to handle tokens or, possibly, even more valuable things. Furthermore, every
-execution of a smart contract happens in public and, in addition to that,
-the source code is often available.
-
-Of course you always have to consider how much is at stake:
-You can compare a smart contract with a web service that is open to the
-public (and thus, also to malicious actors) and perhaps even open source.
-If you only store your grocery list on that web service, you might not have
-to take too much care, but if you manage your bank account using that web service,
-you should be more careful.
-
-This section will list some pitfalls and general security recommendations but
-can, of course, never be complete.  Also, keep in mind that even if your smart
-contract code is bug-free, the compiler or the platform itself might have a
-bug. A list of some publicly known security-relevant bugs of the compiler can
-be found in the :ref:`list of known bugs<known_bugs>`, which is also
-machine-readable. Note that there is a bug bounty program that covers the code
-generator of the Solidity compiler.
-
-As always, with open source documentation, please help us extend this section
-(especially, some examples would not hurt)!
+In Solidity, this is even more important because you can use smart contracts to handle tokens or,
+possibly, even more valuable things.
+Furthermore, every execution of a smart contract happens in public and,
+in addition to that, the source code is often available.
+
+Of course, you always have to consider how much is at stake:
+You can compare a smart contract with a web service that is open to the public
+(and thus, also to malicious actors) and perhaps even open-source.
+If you only store your grocery list on that web service, you might not have to take too much care,
+but if you manage your bank account using that web service, you should be more careful.
+
+This section will list some pitfalls and general security recommendations
+but can, of course, never be complete.
+Also, keep in mind that even if your smart contract code is bug-free,
+the compiler or the platform itself might have a bug.
+A list of some publicly known security-relevant bugs of the compiler can be found
```

Comment:
```suggestion
A list of some publicly known security-relevant compiler bugs can be found
```

Reply:
Compiler bugs=>Result from the compilation.
Bugs of the compiler=>Bugs that exist on the compiler.